### DISCUSSION TITLE
Use Firebase/AnalyticsWithoutAdIdSupport

### DISCUSSION THREAD

**Discussion by ttkien** [invertase/react-native-firebase#5223] [Ideas] [answered]
Firebase iOS has been released version 7.11.0 which added `Added Firebase/AnalyticsWithoutAdIdSupport subspec for developers of apps that aren’t using AdSupport or subject to AppTrackingTransparency.`  Should we provide a new option to choose `Firebase/AnalyticsWithoutAdIdSupport` besides `Firebase/Analytics`

**Comment by mikehardy** [comment, marked as answer]
Hey @ttkien - I started working on it today but it's not quite ready yet. Should be ready soon - I know it's a blocker for certain apps https://github.com/invertase/react-native-firebase/pull/5224

**Reply by ttkien** [reply]
Thank for your quick action @mikehardy

**Reply by mikehardy** [reply]
Release 11.4.1 just went out with the change related to this among other fixes + features. Enjoy! :rocket:

**Reply by mikehardy** [reply]
(consult the new section of the Analytics usage docs to see how to use it - https://rnfirebase.io )
